src/coreclr/vm/methodtable.cpp

```diff
@@ -7388,6 +7435,13 @@ VOID MethodTable::EnsureInstanceActive()
         }
     }
 
+    // The EnsureInstanceActive function may be called during the final stage of assembly load
```

Comment:
How/where does this happen?

Reply:
It happens when a module constructor requires types from the same module as the module constructor's module to be active. I should add that to the comment.